ipaserver/plugins/baseuser.py

```diff
@@ -429,6 +435,41 @@ class baseuser(LDAPObject):
                     'J:', 'K:', 'L:', 'M:', 'N:', 'O:', 'P:', 'Q:', 'R:',
                     'S:', 'T:', 'U:', 'V:', 'W:', 'X:', 'Y:', 'Z:'),
                 ),
+        Int(
+            'ipasubuidnumber?',
+            label=_('SubUID range start'),
+            cli_name='subuid',
+            doc=_('Start value for subordinate user ID (subuid) range'),
+            minvalue=2 ** 31,
+            maxvalue=2 ** 32,
```

Comment:
Use constants instead? SUBID_RANGE_START is one already defined.

Reply:
Good point!